src/GitTfs/Commands/Init.cs

```diff
@@ -32,7 +33,11 @@ public Init(RemoteOptions remoteOptions, InitOptions initOptions, Globals global
 
         public OptionSet OptionSet
         {
-            get { return _initOptions.OptionSet.Merge(_remoteOptions.OptionSet); }
+            get
+            {
+                return _initOptions.OptionSet.Merge(_remoteOptions.OptionSet)
+                    .Add("gitattributes=", "Path to the .gitattributes file to include", v => _gitAttributesFilePath = v);
```

Comment:
This option is similar to `--gitignore`. Can the definition of the option move into `_remoteOptions`?

Reply:
Actually I had exactly the same idea. But the RemoteOptions are used in quite a lot of commands where they are also
participate in the commands usage/help text with the code you quoted, e.g. that merging of the options like
```csharp
return _initOptions.OptionSet.Merge(_remoteOptions.OptionSet)
```
So this would mean that all commands would get that additional commandline parameter `--gitattributes=<file>`
which merge the `_remoteOptions.OptionSet`. From my look, the following commands would receive that additional parameter:
* Bootstrap
* Clone (indirectly via Init)
* Fetch / QuickFetch
* Create  (indirectly via Clone)
* Init 
* ListRemoteBranches
* Subtree (indrectly via Fetch)
* Verify

To me, this new parameter makes only sense for Init and Clone actually a repo is created and we `git add` that new file.
Keep in mind that the --gitignore flag sometimes (e.g. in Fetch case) only participates by adding only ignores for the
current run and not permantely by `git add`ing the given gitignore to the repo. This is different for the gitattribues,
because AFAIK there is no underlying mechanism to temporarirly add attributes without merging .git/info/attributes and the .gitattributes file toegether.

To sum it up, I don't think this should be added to the _remoteOptions